packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -104,6 +106,14 @@ async function createServer(
   }
 }
 
+function deleteDbIfExists(): void {
+  const workspace = vscode.workspace.rootPath;
+  const dbPath = workspace + '/.sfdx/tools/apex.db';
```

Comment:
Does this require changes on the Apex jar or are you just deleting this first to be safe until we fix that?